src/scripts/Northrend/FrozenHalls/PitOfSaron/pit_of_saron.cpp

```diff
@@ -289,6 +289,9 @@ class npc_pos_leader : public CreatureScript
                             for (uint8 i=0; i<2; ++i)
                                 if (Creature* c = pInstance->instance->GetCreature(pInstance->GetData64(DATA_GUARD_1_GUID+i)))
                                     c->CastSpell(c, 70513, false);
+                            for (uint8 i=0; i<2; ++i)
```

Comment:
why don't you use the same for instead create another one? you'll gain minor performances ( for both cases )